addons/mrp/wizard/mrp_production_split.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, Command
+from odoo.tools import float_round, float_compare
+
+
+class MrpProductionSplit(models.TransientModel):
+    _name = 'mrp.production.split'
+    _description = "Wizard to Split Productions"
```

Comment:
Maybe `MrpProductionSplitMulti`, `'mrp.production.split.multi'`, and `"Wizard to Split Multiple Productions"` would make it a little more obvious as to what the difference is between this wizard vs `mrp.production.split.production`?  (if I understood what it's doing correctly)